src/screens/Events/EventsMap.js

```diff
@@ -159,11 +162,24 @@ export default class EventsMap extends Component {
         </MapView>
         <Fab
           style={{ backgroundColor: colors.primary }}
-          onPress={async ()=> {await this.getEvents()}}
+          onPress={async ()=> {await this.props.getEvents()}}
         >
           <Icon name="refresh" />
         </Fab>
       </View>
     );
   }
 }
+
+const mapStateToProps = state => {
+  let storedEvents = state.events.map(event => ({ ...event }));
```

Comment:
Is each event in the array not an object? Also, you could make this a `const` or just inline it on line 177.